src/coreclr/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt

```diff
@@ -152,20 +162,17 @@ implication        ,X86   ,AVX10v1_V512         ,AVX512DQ
 implication        ,X86   ,AVX10v1_V512         ,AVX512DQ_VL
 implication        ,X86   ,AVX10v1_V512         ,AVX512VBMI
 implication        ,X86   ,AVX10v1_V512         ,AVX512VBMI_VL
+
+; These synthetic ISAs need to appear after the core ISAs
```

Comment:
The comment should explain why it is necessary.

Reply:
Fixed.